docs/ert/reference/configuration/magic_strings.rst

```diff
@@ -213,21 +215,21 @@ Experiment dependent variables
 
    Will be replaced by the basename for the ECLIPSE simulations generated by ERT.
    See :ref:`ECLBASE <eclbase>` for more details.
-   For none-Eclipse jobs this will be replaced by the job name (with an iteration identifier)
+   For non-Eclipse jobs this will be replaced by the job name (with an iteration identifier)
```

Comment:
"job name" is correct here and not touched. This is why we are renaming jobs to steps :)